### ISSUE TITLE
[GraphQL error]: Message: Int cannot represent non 32-bit signed integer value: 10834361632

### ISSUE THREAD

**Issue by meligo** [jstefanop/apolloapi-v2#12]
<img width="1096" alt="Screenshot 2024-08-03 at 20 09 20" src="https://github.com/user-attachments/assets/7030e047-8d8b-4bb0-abb2-be6fcc88461e">
 
I'm getting the above error, tried looking into thigs but seem unable to find the part that I need to convert to Float. 

```
"errors": [
        {
            "message": "Int cannot represent non 32-bit signed integer value: 10824605641",
            "locations": [
                {
                    "line": 119,
                    "column": 17
                }
            ],
            "path": [
                "Miner",
                "stats",
                "result",
                "stats",
                0,
                "master",
                "intervals",
                "int_0",
                "errors"
            ]
        }
    ]
```

**Comment by michelem09** [comment]
We fixed this in dev env and we will release the new version soon.
Thanks!

**Comment by anon-crypto-miner** [comment]
> Are the changes in the dev branch? [main...dev](https://github.com/jstefanop/apolloapi-v2/compare/main...dev)

I was able to resolve this error by downloading the MinerStats.js from the dev branch, replacing the existing file, and rebooting the miner.

**Comment by jstefanop** [comment]
thanks for confirming this fix, 2.0.6 will be pushed to production soon
